lang/en/hvp.php

```diff
@@ -64,6 +64,8 @@
 $string['removeoldmobileauthentries'] = 'Remove old H5P mobile auth entries';
 
 // Admin settings.
+$string['backuplibraries'] = 'Include libraries in backup';
+$string['backuplibraries_help'] = 'Excluding libraries from backup can improve performance during backup, but can only be restored to the same site.';
```

Comment:
> but can only be restored to the same site

This might sound better as a non-but statement. e.g.

```
Backups can only be restored to the same site.
```